docs/developer-tools/python.md

```diff
@@ -70,8 +63,9 @@ amadeus = Client(
 
 try:
     response = amadeus.reference_data.locations.get(
-    keyword='LON',
-    subType=Location.AIRPORT)    
+        keyword='LON',
```

Comment:
Indentation to conform with [PEP 8](https://peps.python.org/pep-0008/#indentation)